glusterd2/commands/volumes/volume-create.go

```diff
@@ -57,6 +57,13 @@ func createVolinfo(req *api.VolCreateReq) (*volume.Volinfo, error) {
 		v.ReplicaCount = req.Replica
 	}
 
+	if req.Arbiter != 0 {
```

Comment:
Does it make sense to have all the feature specific validations captured in a function? I'd not like to see the core GD2 functions getting bigger with different validations being added into the code directly. Instead a simple function call would make things more readable IMO.